docs/dev/how-to/send-message-l2-l1.md

```diff
@@ -0,0 +1,286 @@
+# Send an L2 to L1 message
+
+It is impossible to send transactions directly from L2 to L1. 
+
+Instead, you can send arbitrary-length messages from zkSync Era to Ethereum, and then handle the received message on Ethereum with an L1 smart contract.
+
+:::warning What is a message?
+- A message is like an event on Ethereum. 
+- The difference is that a message publishes data on L1. 
+- [Solidity representation](https://github.com/matter-labs/v2-testnet-contracts/blob/b8449bf9c819098cc8bfee0549ff5094456be51d/l1/contracts/zksync/Storage.sol#L58):
+    ```solidity
+    struct L2Message {
+            address sender;
+            bytes data;
+            uint256 txNumberInblock;
+    }
+    ```
+:::
+
+:::tip Verification
+- Verification and confirmation is possible using Ethereum data. 
+- However, zkSync Era has an efficient [request proof function](#prove-the-result) which does the same.
+:::
+
+## Common use cases
+
+Along with zkSync Era's built-in censorship resistance that requires multi-layer interoperability, there are some common use cases that need L2 to L1 transaction functionality, such as:
+
+- Bridging funds from L2 to L1.
+- Layer 2 governance ..
+
+## Send a message
```

Comment:
Is this our canonical format for titles? Not "Sending a message?

Reply:
Imperative for tech docs.